skyportal/handlers/api/summary_query.py

```diff
@@ -122,10 +122,17 @@ def post(self):
           name: q
```

Comment:
maybe we can name this something better than q if we are going to use it as a variable we reference?

Reply:
I wanted to avoid calling it `query` in the openapi spec since there's another meaning of `query`.
```yaml
 - in: query
          name: query
          schema:
              type: string
```
feel awk.